.github/workflows/deploy.yml

```diff
@@ -352,14 +372,36 @@ jobs:
       docker-pass: ${{ secrets.DOCKER_REGISTRY_PASSWORD }}
       slack-token: ${{ secrets.SLACK_BOT_TOKEN}}
 
+  check-openapi-clients:
+    name: Check OpenAPI clients for changes
+    needs:
+      [
+        deploy-staging-nest-forms-backend,
+        deploy-staging-nest-clamav-scanner,
+        deploy-staging-nest-tax-backend,
+        deploy-staging-nest-city-account,
+      ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build OpenAPI client check image
+        working-directory: openapi-clients
+        run: docker build -f Dockerfile.check -t openapi-check .
```

Comment:
(optional) Would prefer to use official Docker GH actions. Or if you don't care about the image you can run it in one command https://stackoverflow.com/a/51314059